adafruit_rfm9x.py

```diff
@@ -646,7 +646,7 @@ def enable_crc(self, val):
                 self._read_u8(_RH_RF95_REG_1E_MODEM_CONFIG2) & 0xfb
             )
 
-    def send(self, data, timeout=2.,
+    def send(self, data, timeout=2., keep_listening=False,
```

Comment:
Could you add a docstring for the `keep_listening` kwarg here?

Reply:
implemented